src/patternfly/components/Card/docs/code.md

```diff
@@ -1,12 +1,13 @@
 ## Overview
 
-A card is a generic rectangular container that can be used to build other components. A card can have three optional sections: header, body, and footer.
+A card is a generic rectangular container that can be used to build other components.
 
 ## Usage
 
-| Class                | Applied     | Outcome                                                                                                                                                                              |
-| -------------------- | ----------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
-| `.pf-c-card`          | `<div>`     | Creates a card containing content.                                                                                                                                                     |
-| `.pf-c-card__header`          | `.pf-c-card`     | Creates the header of a card.                                                                                                                                                     |
-| `.pf-c-card__body`          | `.pf-c-card`     | Creates the body of a card.                                                                                                                                                 |
-| `.pf-c-card__footer`          | `.pf-c-card`     | Creates the footer of a card.                                                                                                                                                     |
+| Class | Applied | Outcome |
+| ---- | ---- | ---- |
+| `.pf-c-card` | `<div>` | Creates a card containing content. **Required** |
+| `.pf-c-card__header` | `.pf-c-card` | Creates the header of a card. |
```

Comment:
Isn't the "Applied" heading for classes the type of html markup it uses, rather then the class it can be use inside of. I thought for modifiers we do that.

Reply:
oh good call! updated.